binaries/conda/torch-model-archiver/meta.yaml

```diff
@@ -14,7 +14,6 @@ requirements:
     - python
     - pillow
     - psutil
-    - future
```

Comment:
can you test that the conda binaries still work by creating them? The pypi ones are tested in CI just not these https://github.com/pytorch/serve/tree/master/binaries/conda

Reply:
Thanks for pointing that out, will test the conda binaries as well.

Reply:
Added details in the summary about conda package testing.